python/cugraph/cugraph/experimental/community/triangle_count.py

```diff
@@ -89,11 +89,11 @@ def EXPERIMENTAL__triangle_count(G, start_list=None):
 
     srcs = G.edgelist.edgelist_df['src']
     dsts = G.edgelist.edgelist_df['dst']
-    weights = G.edgelist.edgelist_df['weights']
 
-    if srcs.dtype != 'int32':
-        raise ValueError(f"Graph vertices must have int32 values, "
-                         f"got: {srcs.dtype}")
+    if ('weights' not in G.edgelist.edgelist_df.columns):
```

Comment:
why not use "is_weighted"?

Reply:
replaced all weight checks with is_weighted